auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -166,6 +166,7 @@ private AuthUI(FirebaseApp app) {
         mAuth = FirebaseAuth.getInstance(mApp);
 
         mAuth.useAppLanguage();
+        mAuth.setFirebaseUIVersion(BuildConfig.VERSION_NAME);
```

Comment:
Is there anything else to this method? I've noticed that it returns a task (should we be listening for something?) and takes in a nullable parameter (is there a set/unset lifecycle?).